FinSearchUnified/Tests/Components/ConfigLoaderTest.php

```diff
@@ -15,7 +15,7 @@ class ConfigLoaderTest extends TestCase
 {
     protected static $ensureLoadedPlugins = [
         'FinSearchUnified' => [
-            'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ'
+            'ShopKey' => '8D6CA2E49FB7CD09889CC0E2929F86B0'
```

Comment:
Never use real existing shopkeys in tests. Syntactically they should look like real shopkeys, but shouldn't really exist. In the worst case they call for example the live service in tests, which is not what we want.

Change the shopkey everywhere you changed it to be a real shopkey.